source/Octopus.Tentacle/Communications/HalibutInitializer.cs

```diff
@@ -100,7 +102,19 @@ void AddPollingEndpoints()
 
                 var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimits();
                 var serviceEndPoint = new ServiceEndPoint(pollingEndPoint.Address, pollingEndPoint.Thumbprint, halibutProxy, halibutTimeoutsAndLimits);
-                halibut.Poll(new Uri(pollingEndPoint.SubscriptionId), serviceEndPoint, CancellationToken.None);
+
+                //Open multiple polling connections if the env var is set to a non-zero/negative number
+                var connectionCount = int.TryParse(Environment.GetEnvironmentVariable(EnvironmentVariables.TentaclePollingConnectionCount), out var count)
+                    ? count
+                    : 1;
+
+                // negative numbers and 0 are coerced to 1
```

Comment:
Nitpick, Would using uint32 help eliminate the need to check for negative values?

Reply:
probably yes. It'd still need to check for 0, so not sure it'll gain much?

Reply:
Changed to `uint`